firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -15,13 +15,21 @@
 package com.google.firebase.installations;
 
 import android.text.TextUtils;
+
+import androidx.annotation.NonNull;
+
 import com.google.firebase.installations.local.PersistedInstallationEntry;
 import java.util.concurrent.TimeUnit;
 
+import com.google.re2j.Pattern;
```

Comment:
a) please no empty line around imports
b) should imports be ordered alphabetically? if yes, please fix ordering